Library/Homebrew/utils/analytics.sh

```diff
@@ -14,17 +22,15 @@ setup-analytics() {
   if [[ "$(git config --file="$HOMEBREW_REPOSITORY/.git/config" --get homebrew.analyticsmessage)" != "true" ||
         "$(git config --file="$HOMEBREW_REPOSITORY/.git/config" --get homebrew.analyticsdisabled)" = "true" ]]
   then
```

Comment:
Actually I left it there intentionally, so that if you reenable analytics the UUID won't change. IMO the old behavior was justifiable because it was an outstanding file; same doesn't hold for new behavior. Of course it is debatable.